notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    prompt = "cowboy"
</pre><p>can we test both pt and ov on the same prompt for making sure that results are the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>